### PR TITLE
sequelize.js

### PR DESCRIPTION
**Pull request by TNThacker2015**

### CODE REVIEW

File: sequelize.js [hunk 1]
```diff
@@ -69,6 +69,10 @@ const WorkerInfo = sequelize.define("workerInfo", {
 		primaryKey: true,
 		allowNull: false
 	},
+	username: {
```
Comment (Yamboy1) [inline sequelize.js]: Why do we need this? we can just access the username from the id

**Comment by Yamboy1** [comment]
Not required, closing

**Comment by TNThacker2015** [comment]
in case someone leaves.

**Comment by TNThacker2015** [comment]
not to mention this also adds an overall and a ratings

**Comment by Yamboy1** [comment]
how useful is that stuff though, when the important stuff isn't added yet
Orders don't persist over restarts, and here you are making a ratings command...

**Comment by Yamboy1** [comment]
plus the overall command isn't necessarily needed, and the ratings one is made badly

**Pull request closed without merging**